oauthenticator/globus.py

```diff
@@ -79,7 +96,7 @@ def _authorize_url_default(self):
     ).tag(config=True)
 
     def _identity_provider_default(self):
-        return os.getenv('IDENTITY_PROVIDER', 'globusid.org')
+        return os.getenv('IDENTITY_PROVIDER', '')
```

Comment:
This would end up in a different PR, due to it being a change to the behavior of the OAuthenticator. I think it generally makes more sense to leave the OAuthenticator open to any provider on initial install.